docs/developers/platform.md

```diff
@@ -45,7 +45,8 @@ graph TD
 
 ### Raw Storage
 
-We save all files
+We first save all raw files before sending to pipeline-runner.
+**Pipeline-runner** is a service that has several functionality to modify data. 
```

Comment:
something like this? - "Pipeline-runner is a service that runs the data package pipelines. It is used to normalise and modify the data before it is displayed publicly"